backend/src/v5/schemas/tickets/templates.constants.js

```diff
@@ -99,22 +101,23 @@ TemplateConstants.presetModulesProperties = {
 
 };
 
-TemplateConstants.defaultProperties = [
+TemplateConstants.defaultProperties = (config) => [
 	createPropertyEntry('Description', propTypes.LONG_TEXT),
 	createPropertyEntry('Owner', propTypes.TEXT, undefined, undefined, true),
 	createPropertyEntry('Created at', propTypes.DATE, undefined, undefined, true),
 	createPropertyEntry('Updated at', propTypes.DATE, undefined, undefined, true),
-	createPropertyEntry('Status', propTypes.ONE_OF, ['Open', 'In Progress', 'For Approval', 'Closed', 'Void'], 'Open'),
+	createPropertyEntry('Status', propTypes.ONE_OF,
+		config?.status ? config?.status.values.map((v) => v.name) : ['Open', 'In Progress', 'For Approval', 'Closed', 'Void'],
+		config?.status ? config?.status.default : 'Open'),
```

Comment:
This can be simplified
```suggestion
		config?.status?.default || 'Open'),
```